src/components/counter.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import {countItems,addItems} from '../actions/actions.js';
+import store from '../store/store.js';
+
+
+const Counter = React.createClass({
+	render:function(){
+		return (
+		<div>
+		<h1> ADDED ITEMS : </h1>
+			<button onClick={store.dispatch.bind(this,countItems(1))}>Shirts</button>
+			<button onClick={store.dispatch.bind(this,countItems(1))}>Shoes</button>
+			<button onClick={store.dispatch.bind(this,countItems(1))}>Hats</button>
+			<button onClick={store.dispatch.bind(this,countItems(1))}>Sunglasses</button>
+
+
+
+		<h1> ITEMS COUNT:</h1> 
+			<h3>Shirts:{store.getState()}</h3>
```

Comment:
you do need to get information from the store, but you need to sift out what exactly you need within the store.